src/vaults/interfaces/token/IHook.sol

```diff
@@ -38,8 +38,8 @@ interface IHook is IERC165 {
         returns (bytes4);
 
     /// @notice Callback on authorized ERC20 transfer.
-    /// @dev    MUST return bytes4(keccak256("onERC20AuthTransfer(address,address,address,uint256,(bytes16,bytes16))"))
-    ///         if successful
+    /// @dev    Cannot be blocked, can only be used to update state.
```

Comment:
I've decided we should just not allow blocking `authTransferFrom`. No real reason to, and it simplifies several paths (removing try-catches)